index.html

```diff
@@ -518,11 +518,12 @@ <h2>Algorithms</h2>
       </p>
 
       <p>
-When the RDF Dataset Canonicalization Algorithm [[RDF-CANON]] is used with ECDSA
-algorithms, the cryptographic hashing function that is passed to the algorithm
-MUST be determined by the size of the associated public key. For P-256 keys,
-SHA-2 with 256 bits of output is utilized. For P-384 keys, SHA-2 with 384-bits
-of output is utilized.
+When the RDF Dataset Canonicalization Algorithm (RDFC-1.0) [[RDF-CANON]] is used
+with ECDSA algorithms, the cryptographic hashing function used by RDFC-1.0 MUST
+be chosen based on the size of the associated public key. For P-256 keys, the
```

Comment:
I would make the following change because the first normative statement doesn't stand on its own, but requires the following two to make any sense/be testable. The only sensible way to test the first MUST is really by testing the subsequent MUSTs, so I'd change the first to an "is".

```suggestion
with ECDSA algorithms, the cryptographic hashing function used by RDFC-1.0 is
be chosen based on the size of the associated public key. For P-256 keys, the
```